core/desugarModules.ml

```diff
@@ -17,328 +24,497 @@
  *
  *  --->
  *
- * val Foo.bobsleigh = ...;
- * fun Foo.x() { ...}
- * fun Foo.Bar.y() { ... }
- * val x = ...;
+ * val Foo0.bobsleigh0 = ...;
```

Comment:
I'm assuming these are internal and aren't exposed like this -- could you modify the comment to make this clearer?

Reply:
Sure!